Userland/Libraries/LibAudio/Resampler.h

```diff
@@ -50,6 +51,21 @@ class ResampleHelper {
         return false;
     }
 
+    template<ArrayLike<SampleType> Samples>
+    Vector<SampleType> resample(Samples to_resample)
```

Comment:
Hmm, can this not lead to copying of `to_resample`?

Reply:
Absolutely, I'll change that to an rvalue reference.